lib/reports/middleware/logging.rb

```diff
@@ -0,0 +1,25 @@
+require "logger"
+
+module Reports
+  module Middleware
+    class Logging < Faraday::Middleware
+      def initialize(app)
+        super(app)
+        @logger = Logger.new(STDOUT)
+        @logger.formatter = proc do |severity, datetime, program, message|
```

Comment:
Unused block argument - `severity`. If it's necessary, use `_` or `_severity` as an argument name to indicate that it won't be used.<br>Unused block argument - `datetime`. If it's necessary, use `_` or `_datetime` as an argument name to indicate that it won't be used.<br>Unused block argument - `program`. If it's necessary, use `_` or `_program` as an argument name to indicate that it won't be used.